Sources/Helpers/OAuthHelper.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  OAuthHelper.swift
+//  OsmAnd Maps
+//
+//  Created by nnngrach on 15.05.2023.
+//  Copyright © 2023 OsmAnd. All rights reserved.
+//
+
+// OAuth version: 2.0
+// Grant type: Authorization code
+
+import SwiftUI
+import Foundation
+import AuthenticationServices
+
+@objc class OAuthHelper : NSObject {
```

Comment:
Use ```@objc(OAOAuthHelper)``` and ```@objcMembers``` at the top of the declaration. This way, the class name will have old naming convention in ObjC and you do not need to add @objc before each method